packages/node/src/Session.ts

```diff
@@ -73,7 +73,7 @@ export interface ISessionOptions {
 /**
  * If no external storage is provided, this storage gets used.
  */
-const defaultStorage = new InMemoryStorage();
+export const defaultStorage = new InMemoryStorage();
```

Comment:
Maybe it's time we add an exports map to SCAN as well, so that at least in Node 12 + (i.e. soon all supported releases) it's not actually possible for developers to reach into the package and grab this.